lxd/storage/drivers/driver_btrfs_volumes.go

```diff
@@ -1272,6 +1272,7 @@ func (d *btrfs) migrateVolumeOptimized(vol Volume, conn io.ReadWriteCloser, volS
 		sentVols := 0
 
 		// Send volume (and any subvolumes if supported) to target.
+		//revive:disable:defer Allow defer inside a loop.
```

Comment:
Similar to previos `error-return` rule. Disabled `defer inside loop` error using a comment. In configuration we can entirely disable "defer inside loop" errors, but not for one case.

Rule docs: https://revive.run/r#defer